features/mbedtls/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/src/default_random_seed.cpp

```diff
@@ -5,7 +5,8 @@
 
 int mbed_default_seed_read(unsigned char *buf, size_t buf_len)
 {
-    psa_status_t rc = psa_its_get(PSA_CRYPTO_ITS_RANDOM_SEED_UID, 0, buf_len, buf);
+    size_t actual_size;
```

Comment:
Should we do anything with `actual_size` after the call to get?

Reply:
Don't think so. We could check that the returned size is the same as the one we expect, but this seems a bit overcautious to me.

Reply:
This is for reading entropy, the foundation upon which all cryptography in the system relies for security. It's worth the caution!

Reply:
OK, good point. Will add the check.

Reply:
Actually, looking at the code, I see that this function is placed at a very deep layer, while the actual entropy algorithm is in a much higher layer. Not sure that how legitimate it is to check for the expected random seed size at this deep layer.

Reply:
To add to that, this is really out of scope for this PR. One could perform this check using the `get_info` API as well. The returned actual size argument from `psa_its_get` is just a more convenient way to achieve this value.